x/concentrated-liquidity/internal/swapstrategy/swap_strategy_test.go

```diff
@@ -39,29 +39,29 @@ func (suite *StrategyTestSuite) TestComputeSwapState() {
 	}{
 		"happy path: trade asset0 for asset1": {
 			sqrtPCurrent:    sdk.MustNewDecFromStr("70.710678118654752440"), // 5000
-			nextSqrtPrice:   sdk.MustNewDecFromStr("70.666662070529219856"), // 4993.777128190373086350
-			liquidity:       sdk.MustNewDecFromStr("1517818840.967515822610790519"),
+			nextSqrtPrice:   sdk.MustNewDecFromStr("70.666662070529219856"), // 4993.7771281903730
```

Comment:
I'm confused why we are changing these. This test does not seem to be touched by the tick logic.

What is the reason? If unrelated, can this be submitted separately?

Reply:
The DefaultLiquidityAmt changed when we changed tick logic 

https://github.com/osmosis-labs/osmosis/blob/e826b666abc42b34f02b32aecde703cda4fcde48/x/concentrated-liquidity/keeper_test.go#L32-L33